tests/unittest/conftest.py

```diff
@@ -48,3 +48,12 @@ def pytest_runtest_protocol(item, nextitem):
 
                 torch.cuda.empty_cache()
             break
+
```

Comment:
Not all unittests are using `pytest`. Some of them are written using the `unittest` framework and are just run via `pytest`. Will this autouse fixture be added also to tests written in `unittest`?